README.md

```
@@ -1,48 +1,104 @@
+### Components 🧩
+Reusable components are in the `(components)` directory, such as:
+- `card.jsx` 🧭
+
+### Global Components 🌍
+Global components are in the root of `app` directory:
+- `layout.jsx` 📐
+- `global.css` 🎨
+
+### Data 📊
+Static content, like projects and skills, is stored in the `data` directory:
+- Projects 🛠️
```

Markdownlint (0.37.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)